src/pages/trip/setting/_components/StampCard.tsx

```
@@ -0,0 +1,210 @@
+                    <>
+                        <Button
+                            isCompleted={completed}
+                            onClick={handleCompletedModalOpen}
+                        />
+                        <ConfirmModal
+                            isOpen={isCompleteModalOpen}
+                            onClose={() => setIsCompletedModalOpen(false)}
+                            onConfirm={handleCompletedStamp}
+                            title="스탬프를 완료할까요?"
+                            children="완료 후에는 다시 수정이 어려워요."
+                        />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)